javatests/arcs/sdk/wasm/CollectionApiTest.kt

```diff
@@ -26,20 +33,34 @@ class CollectionApiTest : AbstractCollectionApiTest() {
                     num = handles.inHandle.size.toDouble()
                 )
                 handles.outHandle.store(stored)
+                handles.ioHandle.store(stored)
             }
             "case3" -> {
                 handles.outHandle.remove(stored)
+                handles.ioHandle.remove(stored)
             }
             "case4" -> {
                 val d1 = CollectionApiTest_OutHandle()
                 val iter = handles.inHandle.iterator()
                 val flg = iter.hasNext()
                 val i1 = iter.next()
-                handles.outHandle.store(d1.copy(
-                    txt = "num: ${i1.num.toInt()}",
-                    num = i1.num.let { it * 2 },
-                    flg = flg
-                ))
+                if (x == 3.0) {
+                    handles.outHandle.store(
+                        d1.copy(
+                            txt = "num: ${i1.num.toInt()}",
+                            num = i1.num * 2.0,
+                            flg = flg
+                        )
+                    )
+                } else {
+                    handles.outHandle.store(
```

Comment:
Is this an error message? If so, might be better to put it in the `errors` handle?

Reply:
Yes, but this particle doesn't have an errors handle.